### PR TITLE
[zh-cn] update 'handling different text directions'

### LINKED ISSUE #23817: “处理不同方向的文本”小节中的翻译问题
### MDN URL

https://developer.mozilla.org/zh-CN/docs/Learn/CSS/Building_blocks/Handling_different_text_directions

### 与此议题相关的小节或标题？

应该使用物理属性还是逻辑属性呢？

### 哪些信息是错误、无用或不完善的？

中文译文：
应该使用物理属性还是逻辑属性呢？
逻辑属性是在物理属性之后出现的，因而最近才开始在浏览器中应用。你可以通过查看 MDN 的属性页面来了解浏览器对逻辑属性的支持情况。如果你并没有应用多种书写模式，那么现在你可能更倾向于使用物理属性，因为这些在你使用弹性布局和网格布局时非常有用。
其中，最后一句与英文原文的意思不符，英文原文指的是使用逻辑属性对弹性布局和网格布局有用。

### 你期望看到什么？

这一段的英文原文为：
Should you use physical or logical properties?
The logical properties and values are newer than their physical equivalents, and therefore have only recently been implemented in browsers. You can check any property page on MDN to see how far back the browser support goes. If you are not using multiple writing modes, then for now you might prefer to use the physical versions. However, ultimately we expect that people will transition to the logical versions for most things, as they make a lot of sense once you also start dealing with layout methods such as flexbox and grid.
个人认为最后一句的意思是：然而，最终我们希望人们将大多数事情转移到逻辑版本，因为它们在你处理布局方法（比如flexbox和grid）时很有用。

### 你有什么支持的链接、参考或引用？

_No response_

### 你还想补充什么内容？

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `zh-cn/learn/css/building_blocks/handling_different_text_directions`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/Learn/CSS/Building_blocks/Handling_different_text_directions
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md
* Last commit: https://github.com/mdn/translated-content/commit/3bd22c6a65f9a77e4cdd95ed4825330df9ddd710
* Document last modified: 2023-09-19T06:05:04.000Z

</details>

### PR DESCRIPTION
**Pull request by jasonren0403**
### Related issues and pull requests

fixes #23817

part of #12632, part of #24474

### CODE REVIEW

File: files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md [hunk 1]
```diff
@@ -1,136 +1,353 @@
...
+### 应该使用物理属性还是逻辑属性呢？
 
-> **备注：** [`float`](/zh-CN/docs/Web/CSS/float)的逻辑值暂时只有 Firefox 和 Firefox for Android 支持，上面的例子可能无法生效。
+逻辑属性和值比物理属性和值更新颖，因此最近才在浏览器中实现。你可以查看 MDN 上的任何属性页面，了解浏览器支持的历史。如果不使用多种书写模式，那么目前可能更愿意使用物理版本。不过，我们预计人们最终会在大多数情况下过渡到逻辑版本，因为一旦开始使用弹性盒和网格等布局方法，逻辑版本就会变得非常有意义。
 
-### 应该使用物理属性还是逻辑属性呢？
+## 技能测试！
 
-逻辑属性是在物理属性之后出现的，因而最近才开始在浏览器中应用。你可以通过查看 MDN 的属性页面来了解浏览器对逻辑属性的支持情况。如果你并没有应用多种书写模式，那么现在你可能更倾向于使用物理属性，因为这些在你使用弹性布局和网格布局时非常有用。
+本文已经结束，但你还记得最重要的信息吗？在继续学习之前，可以找到一些进一步的测试，以验证您是否保留了这些信息——请参阅[技能测试：书写模式和逻辑属性](/zh-CN/docs/Learn/CSS/Building_blocks/Writing_Modes_Tasks)。
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md:345]:
```suggestion
本文已经结束，但你还记得最重要的信息吗？在继续学习之前，可以找到一些进一步的测试，以验证是否保留了这些信息——请参阅[技能测试：书写模式和逻辑属性](/zh-CN/docs/Learn/CSS/Building_blocks/Writing_Modes_Tasks)。
```

File: files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md [hunk 2]
```diff
@@ -1,136 +1,354 @@
...
   </tbody>
 </table>
 
-## 什么是书写模式
+## 什么是书写模式？
 
 CSS 中的书写模式是指文本的排列方向是横向还是纵向的。{{cssxref("writing-mode")}} 属性使我们从一种模式切换到另一种模式。为此，你不必使用一种竖向的语言——你还可以更改部分文字的方向以实现创新性的布局。
 
-下面的例子中，我们使用`writing-mode: vertical-rl`对一个标题的显示进行设置。现在，标题文本是竖向的了。竖向文本在平面设计中很常见，也可以为你的网页设计增添更加有趣的外观。
+在下面的示例中，我们使用 `writing-mode: vertical-rl` 显示了一个标题。现在文本垂直显示。垂直文本在图形设计中很常见，可以为网页设计添加更有趣的外观和感觉。
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md:44]: https://zh.wikipedia.org/zh-cn/%E7%95%8C%E9%9D%A2%E5%A4%96%E8%A7%82

```suggestion
在下面的示例中，我们使用 `writing-mode: vertical-rl` 显示了一个标题。现在文本垂直显示。垂直文本在图形设计中很常见，可以为网页设计添加更有趣的界面外观。
```

File: files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md [hunk 3]
```diff
@@ -1,136 +1,354 @@
...
+}
+
+h2 {
+  border-bottom: 5px solid black;
+}
+```
+
+{{EmbedLiveSample("logical-mbp", "", "200px")}}
+
+如果考虑到所有单独的边框完整表述，属性的数量是非常庞大的，你可以在[逻辑属性和值](/zh-CN/docs/Web/CSS/CSS_logical_properties_and_values) MDN 页面上看到所有映射的属性。
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md:292]:
```suggestion
如果考虑到所有单独的边框完整表述，属性的数量是非常庞大的，你可以在[逻辑属性和逻辑值](/zh-CN/docs/Web/CSS/CSS_logical_properties_and_values) MDN 页面上看到所有映射的属性。
```

File: files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md [hunk 4]
```diff
@@ -1,136 +1,354 @@
...
 
 ### 逻辑值
 
-目前为止我们看到的都是逻辑属性的名称。还有一些属性的取值是一些物理值（如`top`、`right`、`bottom`和`left`）。这些值同样拥有逻辑值映射（`block-start`、`inline-end`、`block-end`和`inline-start`）。
-
-例如，你可以将一张图片移到左边，并使文本环绕图片。你可以将`left`替换为`inline-start` ，就像下面的例子中一样。
+目前为止我们看到的都是逻辑属性的名称。还有一些属性的取值是一些物理值（如`top`、`right`、`bottom` 和 `left`）。这些值同样拥有逻辑值映射（`block-start`、`inline-end`、`block-end` 和 `inline-start`）。
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md:296]:
```suggestion
目前为止我们看到的都是逻辑属性的名称。还有一些属性的取值是一些物理值（如 `top`、`right`、`bottom` 和 `left`）。这些值同样拥有逻辑值映射（`block-start`、`inline-end`、`block-end` 和 `inline-start`）。
```

**Review by yin1999 — APPROVED** [review]
Almost looks good to me. Just some nits:

File: files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md [hunk 5]
```diff
@@ -1,136 +1,354 @@
...
+          >安装了基本的软件</a
+        >，了解基本的<a
+          href="/zh-CN/docs/Learn/Getting_started_with_the_web/Dealing_with_files"
+          >文件管理知识</a
+        >，具有 HTML 基础（参见 <a
           href="/zh-CN/docs/Learn/HTML/Introduction_to_HTML"
-          >Introduction to HTML</a
-        >）以及 CSS 基础（学习<a href="/zh-CN/docs/Learn/CSS/First_steps"
-          >CSS first steps</a
+          >HTML 介绍L</a
```
Comment (jasonren0403) [inline files/zh-cn/learn/css/building_blocks/handling_different_text_directions/index.md:27]:
```suggestion
          >HTML 介绍</a
```

**Pull request merged**